docs/1.getting-started/5.third-parties.md

```diff
@@ -0,0 +1,153 @@
+
+Adding a single third-party resource to your Nuxt application might not have any noticeable impact on performance, but it can quickly begin to affect user experience if you’re not careful. Many third parties, especially scripts, can take a relatively long time to download and execute, which can delay user interactivity and block page rendering.
+
+Data from the Chrome User Experience Report shows that Nuxt sites that load more third-party resources have lower [Interaction to Next Paint](https://web.dev/articles/inp) (INP) and [Largest Contentful Paint](https://web.dev/articles/lcp) (LCP) pass rates.
+
+![Chrome User Experience Report chart](/assets/docs/getting-started/third-parties/chart.png)
+<sub>
+source: Chrome User Experience Report,
+date: October 2023,
+devices: phone
+</sub>
+
+While the chart's correlation does not automatically indicate causation, lab experiments and data from the [Web Almanac](https://almanac.httparchive.org/en/2022/third-parties) provide further evidence that third-party resources significantly affect page performance.
+
+## Optimizing Third Party Performance
+
+In general, it can be difficult to determine the most optimal way to load different third-party resources. Nuxt provides a number of utilities to improve both the developer and user experience of loading third parties.
+
+::callout
+If you're using very popular third-party libraries, please check the [Third-Party Wrappers](#third-party-wrappers) section for available pre-configured solutions.
+::
+
+### useScript
+
+The `useScript` composable enables you to load third-party scripts with SSR support and a proxied API. It works out of the box, requiring just an ID and a URL that points to the script resource.
+
+```vue
+<script setup lang="ts">
+useScript({
+  id: 'some_id',
+  src: 'https://www.example.com/script.js'
+})
+</script>
+```
+
+Optional triggers and asset strategies can be used to allow for more fine-grained control over how, and when, the script should be loaded. In the following example, the script will load once the promise provided in the trigger resolves (1 second later).
+
+```vue
+<script setup lang="ts">
+useScript({
+  id: 'some_id',
+  src: 'https://www.example.com/script.js',
+  trigger: new Promise((resolve) => {
+    setTimeout(() => { resolve() }, 1000)
+  })
+})
+</script>
+```
+
+### useStyles
+
+The `useStyles` composable enables you to leverage `nuxt-assets` for third-party stylesheets.
+
+```vue
+<script setup lang="ts">
+useStyles(['https://www.example.com/styles.css'])
+</script>
+```
+
+You can use optional asset strategies to inline styles directly in the `head` section of your `document`.
+
+```vue
+<script setup lang="ts">
+useStyles(
```

Comment:
The issue with promoting these lower level composables, especially in this example, is that the lower level implementation at some point should be parsing and resolving this stylesheet at _build-time_ instead of the current runtime implementation